graphdatascience/tests/integration/conftest.py

```diff
@@ -44,6 +46,22 @@ def gds() -> GraphDataScience:
     return _gds
 
 
+@pytest.fixture(scope="package")
+def gds_with_tls(request: FixtureRequest) -> GraphDataScience:
```

Comment:
I guess this is ok, but probably cleaner to just use `pathlib` or something to get the path